service/authorization/authorization.go

```diff
@@ -76,20 +99,33 @@ func NewRegistration() serviceregistry.Registration {
 					panic("Error casting clientSecert to string")
 				}
 			}
```

Comment:
NIT: clientSecert has a typo

also i think you want clientSecret and clientId to be lowercase since the config is forced lowercase.

Reply:
thanks good catch! i changed the case in the config but forgot the change it here